### PR TITLE
fix photos.py for python3, add notes support

### PR DESCRIPTION
**Pull request by RubenRocha**

### CODE REVIEW

**Comment by RubenRocha** [comment]
I also just added support for the iCloud notes, and changed uuid1 to uuid4 as it's more secure, but feel free to leave that out.

File: pyicloud/base.py [hunk 1]
```diff
@@ -118,7 +119,7 @@ def __init__(
             password = get_password_from_keyring(apple_id)
 
         self.data = {}
-        self.client_id = str(uuid.uuid1()).upper()
+        self.client_id = str(uuid.uuid4()).upper()
```
Comment (torarnv) [inline pyicloud/base.py]: I'd leave this out, so we can change it globally if it makes sense

File: pyicloud/services/__init__.py [hunk 2]
```diff
@@ -5,3 +5,4 @@
 from pyicloud.services.reminders import RemindersService
 from pyicloud.services.photos import PhotosService
 from pyicloud.services.account import AccountService
+from pyicloud.services.notes import NotesService
```
Comment (torarnv) [inline pyicloud/services/__init__.py]: Add newlines at end of file

File: pyicloud/services/notes.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+			if not note['folderName'] in self.collections:
+				self.collections[note['folderName']] = {}
+			if 'content' in note['detail']:
+				content = self.parseHTML(note['detail']['content'])
+				self.collections[note['folderName']].update({ note['noteId'] : {
+					'dateModified': note['dateModified'],
+					'subject': note['subject'],
+					'size': note['size'], 
+					'content': content
+				}})
```
Comment (torarnv) [inline pyicloud/services/notes.py]: Add newlines at end of file

File: pyicloud/services/notes.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+class NotesService(object):
+	def __init__(self, service_root, session, params):
+		self.session = session
+		self.params = params
+		self._service_root = service_root
+		self.collections = {}
+
+		self.refresh()
+
+	def parseHTML(self, s):
```
Comment (torarnv) [inline pyicloud/services/notes.py]: Is there a standard library or module that provides this already?

File: pyicloud/services/notes.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+		s = s.replace("&amp;", "&")
+		s = re.sub('<br>', "\n", s)
+		s = re.sub('<[A-Za-z\/][^>]*>', '', s)
+
+		return s
+
+	def refresh(self):
+		params_notes = dict(self.params)
+		params_notes.update({
+			'clientVersion': '4.0',
```
Comment (torarnv) [inline pyicloud/services/notes.py]: You don't need to update these each time if they don't change

**Comment by torarnv** [comment]
Hey, thanks for the contribution 👍 

Please squash the pep8 changes of the notes feature into the notes commit.
Also, GitHub is complaining about your email being invalid, please fix that.

Thoughts on why the rest of the pep8 changes are not flagged on our CI or locally?

**Comment by RubenRocha** [comment]
It's all rebased now so hopefully the CI will be able to see the pep8 changes

**Comment by torarnv** [comment]
I added python 3.5 to the CI, and that passed pep8 fine with the current code. Which version are you using? Any idea why your pep8 is complaining and not the one in the CI or the one I'm using?

**Comment by RubenRocha** [comment]
@torarnv I'm on 3.5.2, I'm not too sure why it would be complaining... everything looks fine on my end

**Comment by torarnv** [comment]
Hmm, I need to investigate why you are seeing other pep issues than I do. How are you running pep8? In an IDE?

I'll be gone for four weeks now on vacation, but will continue when I get back.

File: pyicloud/services/notes.py [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+        s = s.replace("&amp;", "&")
+        s = re.sub('<br>', "\n", s)
+        s = re.sub('<[A-Za-z\/][^>]*>', '', s)
+
+        return s
+
+    def refresh(self):
+        params_notes = dict(self.params)
+        params_notes.update({
+            'clientVersion': '4.0',
```
Comment (torarnv) [inline pyicloud/services/notes.py]: No need to update if it doesnt change

File: pyicloud/services/notes.py [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+    def __init__(self, service_root, session, params):
+        self.session = session
+        self.params = params
+        self._service_root = service_root
+        self.collections = {}
+
+        self.refresh()
+
+    def parseHTML(self, s):
```
Comment (torarnv) [inline pyicloud/services/notes.py]: Is there a python function that can do this for us?

**Comment by RubenRocha** [comment]
@torarnv I was using a tool called autopep8, although I did use a switch called --agressive, as for the python function I'm guessing you're talking about stripping the HTML, there are a few ways, although this would add dependencies, such as either lxml, or, beautifulsoup4...

**Comment by torarnv** [comment]
Trying out the new review UI in GitHub 😄 Let me know if you need help splitting out the notes support!

**Comment by cvpe** [comment]
I try to integrate your code into pyicloud.py in IOS but I got an error with 
from tzlocal import get_localzone Which is not found...

**Comment by torarnv** [comment]
No action from contributor, closing.

**Pull request closed without merging**